src/main/java/at/hannibal2/skyhanni/features/misc/customscoreboard/CustomScoreboardUtils.kt

```diff
@@ -0,0 +1,67 @@
+package at.hannibal2.skyhanni.features.misc.customscoreboard
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.HypixelData
+import at.hannibal2.skyhanni.data.ScoreboardData
+import at.hannibal2.skyhanni.mixins.transformers.AccessorGuiPlayerTabOverlay
+import at.hannibal2.skyhanni.utils.RenderUtils.AlignmentEnum
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import net.minecraft.client.Minecraft
+
+private val config get() = SkyHanniMod.feature.gui.customScoreboard
+
+object CustomScoreboardUtils {
+    fun mayorNameToColorCode(input: String): String {
```

Comment:
Feels more consistent in the mayor manager where the api stuff happens. also rename this class then to mayor api